projects/core/components/scroll-controls/scrollbar.directive.ts

```diff
@@ -167,10 +167,12 @@ export class TuiScrollbarDirective {
             : Math.ceil((clientWidth / scrollWidth) * 100) / 100;
     }
 
+    // TODO: change type to `Element | null` for type safety
     private get computedContainer(): Element {
         return this.container
```

Comment:
Looks like we can change it to:
`return this.container?.nativeElement || this.documentRef.documentElement;`

![image](https://user-images.githubusercontent.com/11832552/197494811-875e2076-a601-4cd9-99cd-be94d12c189f.png)

Reply:
fixed